Tests/LibDeviceTree/TestAddressTranslation.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024, Sönke Holz <sholz8530@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibDeviceTree/DeviceTree.h>
+#include <LibDeviceTree/FlattenedDeviceTree.h>
+
+TEST_CASE(address_translation)
+{
+    auto fdt_file = TRY_OR_FAIL(Core::File::open("/usr/Tests/LibDeviceTree/address-translation.dtb"sv, Core::File::OpenMode::Read));
```

Comment:
Can you say in the commit message how you created this .dtb file? (Ideally the command line invocation.)

Reply:
I manually wrote a dts file and compiled it with `dtc -o address-translation.dtb address-translation.dts`.
I could add the source file as well, but I wasn't sure if I should do that.

Reply:
We also could just depend on `dtc` for building serenity and compile it from source, but I don't think we want more build dependencies.

Reply:
Maybe just put the dtc in to the commit message

Reply:
Yeah thats good